docs/logging-usage.rst

```diff
@@ -222,12 +219,43 @@ Add `cloud-logs@google.com` as the owner of `my-bucket-name`:
     >>> from gcloud import storage
     >>> client = storage.Client()
     >>> bucket = client.get_bucket('my-bucket-name')
-    >>> acl = bucket.acl
-    >>> acl.user('cloud-logs@google.com').grant_owner()
-    >>> acl.save()
+    >>> bucket.acl.reload()
+    >>> logs_group = bucket.acl.group('cloud-logs@google.com')
+    >>> logs_group.grant_owner()
+    >>> bucket.acl.add_entity(logs_group)
+    >>> bucket.acl.save()
 
 .. _Set permission for writing exported logs: https://cloud.google.com/logging/docs/export/configure_export#setting_product_name_short_permissions_for_writing_exported_logs
 
+
+Export to BigQuery
+==================
+
+To export logs to BigQuery you must log into the Cloud Platform Console
+and add `cloud-logs@google.com` to your project.
+
+See: `Setting permissions for BigQuery`_
+
+.. _Setting permissions for BigQuery: https://cloud.google.com/logging/docs/export/configure_export#manual-access-bq
+
+
+Export to Pub/Sub
+=================
+
+To export logs to BigQuery you must log into the Cloud Platform Console
+and add `cloud-logs@google.com` to your project.
```

Comment:
Likewise, you can [grant access to the specific topic used by the sink](https://github.com/GoogleCloudPlatform/gcloud-python/blob/b0558ba9060ba0655d35bb90fcc5fe3c00360a3a/system_tests/logging_.py#L235-L241).